backend/src/main/java/ai/verta/modeldb/entities/KeyValueEntity.java

```diff
@@ -29,6 +32,21 @@ public KeyValueEntity() {}
 
   public KeyValueEntity(Object entity, String fieldType, KeyValue keyValue) {
     setKey(keyValue.getKey());
+    if (App.getInstance().mdbConfig.getDatabase().getRdbConfiguration().isMssql()) {
```

Comment:
I noticed that this line is repeated in a couple of places, and some other places have shorter versions, e.g 
`config.getDatabase().getRdbConfiguration().isMssql()`

It would probably be useful to create a single method (perhaps that takes as a parameter either the general config, or perhaps the database returned from `getDatabase()`

Reply:
Extracted duplicate code in util